layers/+lang/extra-langs/packages.el

```diff
@@ -12,6 +15,9 @@
 (defun extra-langs/init-arduino-mode ()
   (use-package arduino-mode :defer t))
 
+(defun extra-langs/init-faust-mode ()
+  (use-package faust-mode :defer t :mode "\\.dsp\\'" :mode "\\.lib\\'"))
```

Comment:
[Here](http://filext.com/file-extension/lib) is a list of `.lib` file types. The only concern I have is for the "common-lisp (library)", but I don't know if it is commonly used or not… maybe someone else knows?